pypeman/nodes.py

```diff
@@ -862,6 +864,29 @@ def process(self, msg):
         return msg
 
 
+class YielderNode(BaseNode):
+    """
+    Take an iterable msg.payload and returns a generator
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def process(self, msg):
+        if not isinstance(msg.payload, collections.Iterable):
+            logger.error("Yielder node took a non iterable msg.payload: %r", msg.payload)
+            raise Rejected()
+
+        def generator(msg):
+            payload = msg.payload
+            ctx = msg.ctx
+            meta = msg.meta
+            for line in payload:
```

Comment:
this is not necessary a `line`.

perhaps `entry` or `item` is more appropriate

Reply:
done